app/src/main/java/org/jellyfin/mobile/player/queue/QueueManager.kt

```diff
@@ -371,9 +372,10 @@ class QueueManager(
     suspend fun selectAudioStreamAndRestartPlayback(stream: MediaStream): Boolean {
         require(stream.type == MediaStreamType.AUDIO)
         val currentMediaSource = currentMediaSourceOrNull ?: return false
+        if (currentMediaSource !is RemoteJellyfinMediaSource) return false
```

Comment:
Why does this have to be a remote source, a download could contain multiple audio tracks?